### PR TITLE
Moved admin music volume slider to its own menu

### LINKED ISSUE #31001: Play Internet Sound is missing a volume specifier
Play Global Sound has one but Play Internet Sound doesn't, normally I like to play my things at like 25-30 volume, can we please get a modified on the server side for Play Internet Sound too?

Thanks

### PR DESCRIPTION
**Pull request by JJRcop**
:cl: JJRcop
tweak: Admin volume slider moved to its own menu for better visibility.
/:cl:

Looks like this:
audio-menu

I don't know what i'm doing.
Resolves #31001

### CODE REVIEW

File: code/modules/goonchat/browserassets/css/browserOutput.css [hunk 1]
```diff
@@ -83,7 +83,7 @@ a.popt {text-decoration: none;}
 #ping {
 	position: fixed;
 	top: 0;
-	right: 40px;
+	right: 80px;
```
Comment (JJRcop) [inline code/modules/goonchat/browserassets/css/browserOutput.css]: I want to point out that this is super dumb, but I'm not very experienced in styling, and my attempts to make it properly follow the other elements didn't work.

**Comment by optimumtact** [comment]
Why does it not look like a slider?

**Comment by JJRcop** [comment]
It still keeps its original functionality of revealing the slider on mouseover, I could change it.

**Comment by optimumtact** [comment]
Oh, I don't think I'm understanding how it works.

I was thinking about doing a remap of the top part of the chatbar anyway so it would be

```
[------------chat menu bar--------]
[---------volume slider for midi--]
Chat scrolling by
```
That way the volume slider is super obvious

**Comment by JJRcop** [comment]
That would be way, way better. I am not very experienced in HTML/JS/CSS, and this has jQuery on top of that which I'm even less experienced in.

Right now I just took the current slider out of the options menu, made a new audio menu, and put the slider in there.

**Comment by optimumtact** [comment]
this pr should be fine, I have a slow schedule so waiting for me would not be appropriate

**Pull request merged**